src/pcf_flags.h

```diff
@@ -11,12 +11,17 @@
 #include "enum_flags.h"
 #include "logger.h"
 
+
 //// and the ever-so-important array size macro
+
+
 //#ifndef ARRAY_SIZE
+
+
 //#define ARRAY_SIZE(x)    (sizeof(x) / sizeof((x)[0]))
-//#endif
 
-using namespace std;
+
+//#endif
```

Comment:
new lines need to be added to make the format test pass. Will remove in another PR.